platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/InputValidation.java

```diff
@@ -31,14 +31,22 @@
     private static final String DEFAULT_FILE = "file" + DEFAULT_EXTENSION;
 
     private static final List<String> BAD_FILES = Arrays.asList(System.getProperty("bad.files")
+            .toLowerCase()
```

Comment:
Yes. If there is capitalization in the bad.files/bad.file.extensions, they wouldn't have been matched properly.

e.g. a filename as `thumbs.db` would not be matched with `Thumbs.db` in system.properties.

Although I suppose using regex would be the cleanest solution. It's just that the legacy code identified them by converting filenames to lowercase.